gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -195,28 +247,48 @@ private void benchmarkRead(
                 LongSummaryStatistics readCallTimeNs = newLongSummaryStatistics(readCallTimeNsList);
 
                 byte[] readBuffer = new byte[readSize];
+                Path file_to_io = testFile;
+
+                if (!benchmarkRead) {
```

Comment:
instead of not read, make it explicit for write, this allows extensions to benchmarks in the future without side effects

Reply:
replaced boolean with enum. This is more descriptive